tests/e2e_suite.py

```diff
@@ -22,6 +24,11 @@ class TestStatus(Enum):
 
 def run(args):
 
+    if os.getenv("SHUFFLE_SUITE"):
```

Comment:
Could we add an argument to the suite to accept a given seed (so that it's easy to re-run the same suite in case of errors)?

Reply:
Sure, I'll make it an env var also, because the sheer amount of stuff to change in pipelines yaml, cmake and then python to expose generic args is too much for such a niche switch